### ISSUE TITLE
[BUG] Paperless-ngx 2.0.0 upload error: ocrmypdf.exceptions.ColorConversionNeededError: The input PDF has an unusual color space

### ISSUE THREAD

**Issue by mdangst** [paperless-ngx/paperless-ngx#4702]
### Description

After the update from paperless-ngx 1.17.4 to 2.0.0 the PDF upload failed. The following error message was shown:
![image](https://github.com/paperless-ngx/paperless-ngx/assets/118683900/5e045bc1-ee57-435f-b2bb-036d073b741e)
I never had this issue before with version 1.17.4 and in the past documents created from the same company have been uploaded without any issues.

### Steps to reproduce

1. PDF selected for the upload
2. Paperless start with processing the PDF document
3. Then after a few second the process stopped and the error message below was shown
4. 
![image](https://github.com/paperless-ngx/paperless-ngx/assets/118683900/4f696a27-ae51-434e-b60d-02b474bdc080)

### Webserver logs

```bash
File "/usr/local/lib/python3.11/site-packages/ocrmypdf/builtin_plugins/ghostscript.py", line 119, in generate_pdfa

    ghostscript.generate_pdfa(

  File "/usr/local/lib/python3.11/site-packages/ocrmypdf/_exec/ghostscript.py", line 296, in generate_pdfa

    raise ColorConversionNeededError()

ocrmypdf.exceptions.ColorConversionNeededError: The input PDF has an unusual color space. Use

--color-conversion-strategy to convert to a common color space

such as RGB, or use --output-type pdf to skip PDF/A conversion

and retain the original color space.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):

  File "/usr/local/lib/python3.11/site-packages/asgiref/sync.py", line 349, in main_wrap

    raise exc_info[1]

  File "/usr/src/paperless/src/documents/consumer.py", line 441, in try_consume_file

    document_parser.parse(self.path, mime_type, self.filename)

  File "/usr/src/paperless/src/paperless_tesseract/parsers.py", line 388, in parse

    raise ParseError(f"{e.__class__.__name__}: {e!s}") from e

documents.parsers.ParseError: ColorConversionNeededError: The input PDF has an unusual color space. Use

--color-conversion-strategy to convert to a common color space

such as RGB, or use --output-type pdf to skip PDF/A conversion

and retain the original color space
```

### Paperless-ngx version

2.0.0

### Host OS

Synology DMS-Version: DSM 7.2.1-69057 Update 3

### Installation method

Docker - official image

### Browser

Chrome

### Configuration changes

Non

**Comment by shamoon** [comment]
As you can tell, that is being raised by the tool we use to process PDFs, we cant really control the changes they make. On the paperless side you'd be able to affect this with https://docs.paperless-ngx.com/configuration/#PAPERLESS_OCR_USER_ARGS using the setting the output references

**Comment by mdangst** [comment]
Thanks for your swift reply. Don't understand your answer. And I've no idea how to change this settings. Can you help me on this? Is it possible to get back the previous version?

**Comment by shamoon** [comment]
If many users are running into this we may want to look into further

**Comment by stumpylog** [comment]
Looks like it was added [here](https://github.com/ocrmypdf/OCRmyPDF/blob/main/docs/release_notes.rst#v1542).  Pretty slim.  And the command doesn't actually say what the options even are to provide for `--color-conversion-strategy`.

**Comment by shamoon** [comment]
Even something like

```py
        if settings.OCR_OUTPUT_TYPE == "pdfa":
            ocrmypdf_args["color-conversion-strategy"] = "LeaveColorUnchanged"
```

**Comment by stumpylog** [comment]
They're defined here: https://github.com/ocrmypdf/OCRmyPDF/blob/9898904be70bf957154cc66cb5d6f57bac7c26b4/src/ocrmypdf/_exec/ghostscript.py#L24-L32

I don't know that we can choose the best option for the output, given that possible set of options

**Comment by stumpylog** [comment]
Using the sample from #4707, setting that as a user arg isn't enough, we'd need to directly set something like "RGB" to actually convert to something.  But then we're making a decision which might not be the best.  Ugh, PDFs are annoying.

**Comment by stumpylog** [comment]
The output when choosing RGB does look alright though

**Comment by shamoon** [comment]
Ah, bummer. So two options:

1. We leave it to the user to set just like other args.
2. We use RGB and let users override that.

I think 2 would be fine, we've always had users running into color conversion quirks, in some ways that would still be more control than currently. Cant say that I really know how significant the ramifications of it would be

**Comment by mdangst** [comment]
I saw there is another user with having the same issue, see the link belwo:
https://github.com/paperless-ngx/paperless-ngx/issues/4707
Again with 1.17.4 all worked well and with 2.0.0 we have got this issue. Could be by accident but maybe something has changed with version 2.0.0 creating the issue.
